JordanRaleigh/AchievementBot
# Dynamic user functionality

## Description

This code adds the functionality to grab all achievements under your email once you login. This is done by setting the cookie when you login, then grabbing that cookie as a serversideprop and using it in the API GET request. 

I also added Material UI to start on styling

## Type of Changes



| Type                       |
| -------------------------- |
| :sparkles: New feature     |

## Before / Current Behavior
Before the code was limited to a single user and stale username to pull back achievements.

## After / New Behavior
Now, any email that logs in will be able to Get all achievements under their email as the username. 

## Testing Steps / QA Criteria
1. Click Login and go through Google Oauth
2. Refresh and see all achievements pull back
